src/components/core/constants.js

```diff
@@ -1,3 +1,7 @@
+import React from "react";
+import { NavLink } from "react-router-dom";
+import s from "./constants.module.css";
```

Comment:
These imports are not needed here anymore.

Reply:
done